File: fibonacci.c
#include <stdio.h>
#include <stdlib.h>

int main(int argc, char *argv[])
{
    int n = atoi(argv[1]);
    fib(n);
    return 0;
}

int fib(int n)
{
    if(n == 0){
        return 0;
    }else if(n == 1){
        return 1;
    }else{
        return fib(n - 2) + fib(n - 1);
    }
}

File: fibonacci.py
# -*- coding:utf-8 -*-
import sys

def fib(n):
    if n == 0:
        return 0
    if n == 1:
        return 1
    return fib(n - 2) + fib(n - 1)


if __name__ == '__main__':
    n = int(sys.argv[1])
    fib(n)
